pkgs/applications/misc/phoc/default.nix

```diff
@@ -0,0 +1,67 @@
+{ stdenv
+, fetchFromGitLab
+, meson
+, ninja
+, pkgconfig
+, wrapGAppsHook
+, libinput
+, gnome3
+, glib
+, gtk3
+, wayland
+, dbus
+, cmake
+, libdrm
+, libxkbcommon
+, wlroots
+}:
+
+stdenv.mkDerivation rec {
+  pname = "phoc";
+  version = "0.1.7";
+
+  src = fetchFromGitLab {
+    domain = "source.puri.sm";
+    owner = "Librem5";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1007kxap87h1bi79sz45lfb478d93pq0676cr5rnhbanljm1n28n";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkgconfig
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    libdrm.dev
+    libxkbcommon
+    libinput
+    glib
+    gtk3
+    gnome3.gnome-desktop
+    gnome3.mutter
+    wayland
+    wlroots
+  ];
+
+  NIX_CFLAGS_COMPILE = "-I${libdrm.dev}/include/libdrm/";
+
+  dontUseCmakeConfigure = true;
+
+  mesonFlags = ["-Dembed-wlroots=disabled"];
```

Comment:
I think we need to report this upstream as improper use of meson features. Either they should switch to a boolean, or use the opposite (like `system-wlroots`) so that `-Dauto_features=enabled` works as excepted. https://mesonbuild.com/Build-options.html#features

Reply:
I'm sorry but I don't have experience with meson and I don't feel comfortable opening an issue requesting this change.

Reply:
If you open an issue in nixpkgs and cc me, I can do it when I have some time.

Reply:
done: https://github.com/NixOS/nixpkgs/issues/91181